config/prod.exs

```diff
@@ -23,3 +23,16 @@ config :ehmon, :report_mf, {:ehmon, :info_report}
 # which by default uses df flags that aren't available on alpine's busybox.
 # this tells disksup to use different df flags
 config :os_mon, disksup_posix_only: true
+
+# Auth
+config :ueberauth, Ueberauth,
```

Comment:
Does `config/prod.exs` need `config :orbit, OrbitWeb.Auth.Guardian` or `config :ueberauth_oidcc` like `dev.exs` and `test.exs` have?

Reply:
Oh, I think these need to be in `runtime.exs`, which is missing from this PR.

Reply:
👍 fe54917